moveit_ros/benchmarks/scripts/moveit_benchmark_statistics.py

```diff
@@ -342,7 +348,7 @@ def plotAttribute(cur, planners, attribute, typename):
         maxy = max([max(y) for y in measurements])
         for i in range(len(labels)):
             x = i+width/2 if typename=='BOOLEAN' else i+1
-            ax.text(x, .95*maxy, str(nanCounts[i]), horizontalalignment='center', size='small')
+            ### ax.text(x, .95*maxy, str(nanCounts[i]), horizontalalignment='center', size='small')
```

Comment:
Could you please add a comment above the commented line that explains what this would add if uncommented?

Reply:
Will be done.

Reply:
I made your changes in the second commit, please check.
Thanks,